common/workunit/workunit.cpp

```diff
@@ -1494,7 +1494,14 @@ class GraphScopeIterator : public CInterfaceOf<IConstWUScopeIterator>
         {
             if (whichProperties & PTattributes)
             {
-                playAttribute(visitor, WaLabel);
+                Owned<IAttributeIterator> nodeAttrs = cur.getAttributes();
```

Comment:
Doesn't it make more sense to have  isBalanced as an att element like other properties of the activity?

Reply:
It would, but that format is has a much higher memory footprint than needed.  I wish 90% of the attributes were stored as attributes instead of att tags.

I can change for consistency since we're probably never going to replace the format or if we did it would not be piecemeal.